src/waveform/renderers/waveformmark.h

```diff
@@ -94,23 +104,29 @@ class WaveformMark {
     // Check if a point (in image coordinates) lies on drawn image.
     bool contains(QPoint point, Qt::Orientation orientation) const;
 
+    QImage generateImage(float breath, float devicePixelRatio);
+
     QColor m_textColor;
     QString m_text;
     Qt::Alignment m_align;
     QString m_pixmapPath;
+    QString m_iconPath;
 
-    float m_linePosition;
+    float m_linePosition{};
```

Comment:
I'm not sure we use this style of initialization elsewhere, probably clearer to do = 0; instead

Reply:
I would like to get developer concensus on this. @Swiftb0y (IIRC) recently told me to initialize in the initializer list...

But there are two issues at hand here: 1) using curly braces aka uniform initialization and 2) in-class initialization vs initializer lists.

As for 1, from what I understand, uniform initialization is the prefered way since c++11. The standard https://isocpp.org/wiki/faq/cpp11-language#uniform-init clearly says: "Prefer initializing using {}."

As for 2, I don't think there is such a clear preference. When you have multiple constructors, in-class initialization certainly makes things easier, otherwise I don't think it matters much.

Reply:
> I would like to get developer concensus on this. @Swiftb0y (IIRC) recently told me to initialize in the initializer list...

Let me be clear, I didn't mean to tell you to use initializer lists, I just said other people will complain and I thought I'd mention it so it doesn't seem like I didn't review your PR thoroughly. I also prefer this style and if other devs do so too, then lets use it. If others complain we can settle this in a simple vote. 

> I'm not sure we use this style of initialization elsewhere, probably clearer to do = 0; instead


I also prefer `= 0` but only when the type is clearly numerical. On the other hand the meaning of `= 0` is already overloaded in the context of pure virtual member functions. So my preference is probably wrong and we should prefer using `{}` as per the cpp core guidelines.

Reply:
Ok, let's take this discussion to zulip

Reply:
removing in-class initialization